recipes/paraview/v1/gh200/environments.yaml

```diff
@@ -0,0 +1,62 @@
+nvidia:
+  compiler:
+      - toolchain: gcc
+        spec: gcc
+      - toolchain: llvm
+        spec: nvhpc
+  unify: when_possible
+  mpi:
+      spec: cray-mpich@8.1.29%nvhpc
+      gpu: cuda
+  specs:
+  - boost%gcc ~mpi
+  - python@3.10%gcc
+  #- eccodes@2.25.0%nvhpc +tools +fortran +aec +openmp jp2k=jasper
+  - cmake%gcc
+  - cuda@12.3%gcc
+  - hdf5%gcc
+  - hwloc%gcc
+  - netcdf-c%gcc
+  - netcdf-cxx4%gcc
+  - netcdf-fortran%nvhpc
+  - numactl%gcc
+  - osu-micro-benchmarks@5.9%nvhpc
+  # everything needed for nccl on SS11
+  - aws-ofi-nccl@master%gcc
+  - nccl%gcc
+  - nccl-tests%gcc
+  # The following are required to stop spack from using nvhpc to build
+  # basic dependencies, some of which don't compile with nvc etc.
+  # Explicitly excluded as modules.
+  - autoconf%gcc
+  - automake%gcc
+  - ca-certificates-mozilla%gcc
+  - diffutils%gcc
+  - gnuconfig%gcc
+  - libiconv%gcc
+  - libxcrypt%gcc
+  - libxml2%gcc
+  - m4%gcc
+  - ncurses%gcc
+  - openssl%gcc
+  - perl%gcc
+  - xz%gcc
+  - zlib%gcc
+  - zstd%gcc
+  - c-blosc%gcc
+  - libaec%gcc
+  - jasper%gcc
+  - patchelf%gcc
+  - gmake%gcc
+  - paraview@5.12%gcc +mpi +tbb +egl ~qt +fortran +python +hdf5 +nvindex +cdi +raytracing
+  - llvm~cuda
```

Comment:
This is a workaround, because of `cuda` variant which has been set as obsolete starting from `llvm@15` by means of a conflict.

https://github.com/spack/spack/blob/e9149cfc3cfc94bd66aa0e0901ce562cf8b75713/var/spack/repos/builtin/packages/llvm/package.py#L380

This was constraining the concretization to `llvm@14`, which does not support Santis new CUDA architecture.

Reply:
my cmake-based compilation does not need, or use any llvm. So, there must be something [un-necessary] in the way your spack compilation goes. John Biddiscombe would know more. Please ping him.

Reply:
`llvm` is a dependency of `ispc`, which is itself a dependency of `ospray` and `openimagedenoise`.

Reply:
oh yeah, I forgot that one.